src/utils/src/arg_parser.rs

```diff
@@ -11,6 +11,7 @@ pub type Result<T> = result::Result<T, Error>;
 const ARG_PREFIX: &str = "--";
 const ARG_SEPARATOR: &str = "--";
 const HELP_ARG: &str = "--help";
+const SNAPSHOT_VERSION_ARG: &str = "--snapshot-version";
```

Comment:
I would prefer to output the snapshot version information on the second line after the Firecracker version. No need for additional CLI parameter.

Reply:
Ack